drivers/watchdog/wdt_nrfx.c

```diff
@@ -7,9 +7,10 @@
 #include <nrfx_wdt.h>
 #include <watchdog.h>
 
-#define SYS_LOG_DOMAIN "wdt_nrfx"
+#define LOG_MODULE_NAME wdt_nrfx
 #define SYS_LOG_LEVEL CONFIG_SYS_LOG_WDT_LEVEL
```

Comment:
I opened similar PRs for stm32 I2C (#9385) and usb (#9384) drivers. I had to change this line to make it work with the new LOG subsys. Did it work without changing this line?